src/components/moderation/ReportDialog/index.tsx

```diff
@@ -0,0 +1,652 @@
+          return subjectTypes.includes('chat')
+        } else {
+          return subjectTypes.includes('record')
+        }
+      })
+      .filter(l => {
+        const collections: string[] | undefined = l.subjectCollections
+        if (collections === undefined) return true
+        // all chat collections accepted, since only Bluesky handles chats
+        if (props.subject.type === 'chatMessage') return true
+        return collections.includes(props.subject.nsid)
+      })
+      .filter(l => {
+        if (!state.selectedOption) return true
+        const reasonTypes: string[] | undefined = l.reasonTypes
+        if (reasonTypes === undefined) return true
+        return reasonTypes.includes(state.selectedOption.reason)
+      })
+  }, [props, allLabelers, state.selectedOption])
+  const hasSupportedLabelers = !!supportedLabelers.length
+  const hasSingleSupportedLabeler = supportedLabelers.length === 1
+
+  const onSubmit = React.useCallback(async () => {
+    dispatch({type: 'clearError'})
+
+    try {
+      setPending(true)
+      // wait at least 1s, make it feel substantial
+      await wait(
+        1e3,
+        submitReport({
+          subject: props.subject,
+          state,
+        }),
+      )
+      setSuccess(true)
+      // give time for user feedback
+      setTimeout(() => {
+        props.control.close()
+      }, 1e3)
+    } catch (e: any) {
+      logger.error(e, {
+        source: 'ReportDialog',
+      })
+      dispatch({
+        type: 'setError',
+        error: _(msg`Something went wrong. Please try again.`),
+      })
+    } finally {
+      setPending(false)
+    }
+  }, [_, submitReport, state, dispatch, props, setPending, setSuccess])
+
+  return (
+    <Dialog.ScrollableInner
+      label={_(msg`Report dialog`)}
+      ref={ref}
+      style={[a.w_full, {maxWidth: 500}]}>
+      <View style={[a.gap_2xl, isNative && a.pt_md]}>
+        <StepOuter>
+          <StepTitle
+            index={1}
+            title={copy.subtitle}
+            activeIndex1={state.activeStepIndex1}
+          />
+          {isLoading ? (
+            <View style={[a.gap_sm]}>
+              <OptionCardSkeleton />
+              <OptionCardSkeleton />
+              <OptionCardSkeleton />
+              <OptionCardSkeleton />
+              <OptionCardSkeleton />
+              {/* Here to capture focus for a hot sec to prevent flash */}
+              <Pressable accessible={false} />
+            </View>
+          ) : labelersLoadError || !allLabelers ? (
+            <Admonition.Outer type="error">
+              <Admonition.Row>
+                <Admonition.Icon />
+                <Admonition.Text>
+                  <Trans>Something went wrong, please try again</Trans>
+                </Admonition.Text>
+                <Admonition.Button
+                  label={_(msg`Retry loading report options`)}
+                  onPress={() => refetchLabelers()}>
+                  <ButtonText>Retry</ButtonText>
```

Comment:
```suggestion
                  <ButtonText>
                    <Trans>Retry</Trans>
                  </ButtonText>
```
I think translation tags are missing here (not sure if the formatting is right in my suggestion though)

Reply:
Ah good call. Did it locally to ensure formatting, thanks for the bump!